test/onnx/onnx_test_common.py

```diff
@@ -448,7 +451,13 @@ def _compare_pytorch_onnx_with_ort(
     ref_outputs = onnx_program.adapt_torch_outputs_to_onnx(
         ref_model(*ref_input_args, **ref_input_kwargs)
     )
-    ort_outputs = run_ort(onnx_program, onnx_format_args)
+
+    if model_type == TorchModelType.TORCH_EXPORT_EXPORTEDPROGRAM:
+        ort_outputs = onnx_program(*input_args, **input_kwargs)
+    elif model_type == TorchModelType.TORCH_NN_MODULE:
+        ort_outputs = run_ort(onnx_program, onnx_format_args)
```

Comment:
this distinction feels unintuitive. `onnx_program` has type ONNXProgram, why do we need to branch based on another argument `model_type` and not running `__call__` in all cases?

Reply:
Indeed we could use always use `__call__`.

I kept this for `ExportedProgram` only just to test both execution paths, with and without `__call__`, but I am ok in using only `__call__`

Reply:
changed the code to always use __call__